content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,20 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
+link:/doc/book/managing/plugins/[The Plugin Manager tool] downloads plugins and their dependencies into a folder, which enables easier import into an instance of Jenkins.
+This avoids the recreation of docker files, or other implications of plugin management in Jenkins.
+This also provides more information about the plugins being downloaded, such as available updates or security warnings.link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin installation manager tool for Jenkins]. 
```

Comment:
```suggestion
The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] downloads plugins and their dependencies into a folder, which enables easier import into an instance of Jenkins.
This avoids the recreation of docker files, or other implications of plugin management in Jenkins.
This also provides more information about the plugins being downloaded, such as available updates or security warnings. 
```
Moved link to plugin manager introduction line, I think it makes more sense there. The existing link was not spaced away from the text, so moving it also helps sort the text overall.